tests/garage/experiment/test_snapshotter.py

```diff
@@ -19,41 +24,78 @@
 }), ('none', {})]
 
 
-class TestSnapshotter:
+def sample_task(snapshot_config):
```

Comment:
is there a way to make this task lighter? perhaps with a simple fixture or a mock?